db/data_schema.rb

```diff
@@ -1,2 +1,2 @@
 # encoding: UTF-8
-DataMigrate::Data.define(version: 20210202104522)
+DataMigrate::Data.define(version: 20210216140931)
```

Comment:
I am interested to know how we will approach this need to update the data in code if we do decide to get rid of the data migrate gem?

Reply:
I have some Opinons on that, based on how we did it on Claim

Reply:
@pezholio I like opinions!